webapp/src/Entity/Contest.php

```diff
@@ -314,6 +314,15 @@ class Contest extends BaseApiEntity implements AssetEntityInterface
      */
     private bool $openToAllTeams = true;
 
+    /**
+     * @ORM\Column(type="boolean", name="is_locked",
+     *     options={"comment"="Is this contest locked for modifications?",
+     *              "default"=0},
+     *     nullable=false)
+     * @Serializer\Exclude()
```

Comment:
I might be wrong, but I think if we exclude the serializer it does not show up in the API response. I think we want it to show up there.

Reply:
I have followed the other attributes, e.g. public, enabled, openToAllTeams.